main.js

```diff
@@ -11,28 +11,52 @@ function generateNumbers() {
     return allNumbers;
 }
 
-function displayNumbers(allNumbers) {
-    const numbersOnScreen = allNumbers.length;
+function displayNumbers() {
+    const randomNumbers = generateNumbers();
```

Comment:
why are you calling this inside the function? calling it outside and passing the numbers in is a much better solution. avoid calling functions inside other functions as much as possible